packages/@okta/courage/src/vendor/plugins/jquery.autosuggest-1.4.min.js

```diff
@@ -336,7 +336,8 @@ define(['jquery'], function ($) {
 								formatted = opts.formatList.call(this, this_data, formatted);	
 							}
 							results_ul.append(formatted);
-							delete this_data;
+							//delete this_data;
```

Comment:
This was needed to get rid of this error message. 
ERROR in ./target/js/vendor/plugins/jquery.autosuggest-1.4.min.js
Module build failed: SyntaxError: Deleting local variable in strict mode (341:7)

  339 | 							//delete this_data;
  340 | 							//this_data = null;
> 341 | 							delete this_data;
      | 							^
  342 | 							matchCount++;
  343 | 							if(opts.retrieveLimit && opts.retrieveLimit == matchCount ){ break; }
  344 | 						}

 @ ./target/js/shared/views/forms/inputs/BasePicker.js 3:0-466:2

Reply:
Is the change just in this file or is it in courage as well? Should probably address in courage (either by updating it to this code (with a comment why) or disable the linting for that line).

Or, actually, we should probably just ignore linting on vendor files.

Reply:
I agree that this should be done in courage. Not sure why this error is not thrown in courage.

Reply:
It's being ignored:
https://github.com/okta/courage/blob/master/.eslintignore#L1

Reply:
Should we do the same in sign in widget by adding vendor files to the eslintignore list ?

Reply:
Yeah that sounds fine.

Reply:
Tried adding a eslintignore file. That doesn't seem to fix the problem. Seems like a webpack config issue.

Reply:
Reverting this commit https://github.com/okta/okta-signin-widget/commit/af51f2c73279139c430e4e6c8a4071f66cbe8748 got rid of the error. Might have to ignore this file via Balel's ignore list